gpflow/conditionals/mo_conditionals.py

```diff
@@ -75,8 +75,7 @@ def shared_independent_conditional(
     """
     Kmm = covariances.Kuu(inducing_variable, kernel, jitter=default_jitter())  # [M, M]
     Kmn = covariances.Kuf(inducing_variable, kernel, Xnew)  # [M, N]
-    Knn = kernel(Xnew, full_cov=full_cov, full_output_cov=False)
-    Knn = Knn[0, ...] if full_cov else Knn[..., 0]  # [N, N] or [N]
+    Knn = kernel.kernel(Xnew, full_cov=full_cov)
```

Comment:
why this change?

Reply:
because the code was wrong and this was spotted by the asserts

Reply:
This was covered by previous review comments